spec/requests/spaces_controller_request_spec.rb

```diff
@@ -110,4 +111,24 @@
       end
     end
   end
+
+  describe "#new" do
+    subject(:result) do
+      sign_in(nil, user)
+      get polymorphic_path([:new, :space])
+      response
+    end
+
+    context "when not logged in" do
+      let(:user) { nil }
+
+      it { is_expected.to be_not_found }
+    end
+
+    context "when an Operator" do
+      let(:user) { create(:person, operator: true) }
+
+      it { is_expected.to be_ok }
```

Comment:
so only `operators` can make spaces but not space members?

Reply:
Yes, for now. We may want to set up `Neighorhood#accepting_new_spaces?` or something so that `Operator`s can turn self-registration on and off; similar to how Mastodon allows their moderators to enable and disable public registrations.